pkg/database/handler.go

```diff
@@ -221,6 +228,123 @@ func (h Handler) SaveAs(c *gin.Context) {
 	c.JSON(http.StatusCreated, save)
 }
 
+// Save database
+func (h Handler) Save(c *gin.Context) {
+	// swagger:route POST /databases/save/{instanceId} saveDatabase
+	//
+	// Save database
+	//
+	// Saving a database involves the following steps
+	//
+	// 1. Lock database if not already locked
+	// 2. Save as
+	// 3. Delete the existing database
+	// 4. Rename the saved database in the file storage
+	// 5. Overwrite the saved as database's properties with the ones from the old database (including the id)
+	// 6. Unlock the database if it wasn't already locked
+	//
+	// This won't affect the instance running the database. However, it should be noted that if two databases are deployed from the same database they can both overwrite it. It's up to the users to ensure this doesn't happen accidentally.
+	//
+	// Security:
+	//	oauth2:
+	//
+	// Responses:
+	//	202:
+	//	401: Error
+	//	403: Error
+	//	404: Error
+	//	415: Error
+	instanceIdParam := c.Param("instanceId")
+	instanceId, err := strconv.ParseUint(instanceIdParam, 10, 32)
+	if err != nil {
+		badRequest := apperror.NewBadRequest("error parsing instanceId")
```

Comment:
Now that the user service merged with the manager we should use the same error handling approach

https://github.com/dhis2-sre/im-manager/blob/31539cd12138b81a4d6243a1ac10a29ac71664fd/pkg/user/handler.go#L232-L233